app/models/student_loans_data_importer.rb

```diff
@@ -28,12 +28,28 @@ def row_to_hash(row)
       full_name: row.fetch("Full name"),
       date_of_birth: cast_as_date(row.fetch("Date of birth")),
       policy_name: row.fetch("Policy name"),
-      no_of_plans_currently_repaying: row.fetch("No of Plans Currently Repaying"),
-      plan_type_of_deduction: row.fetch("Plan Type of Deduction"),
+      no_of_plans_currently_repaying: calculate_no_of_plans_currently_repaying(row.fetch("No of Plans Currently Repaying")),
+      plan_type_of_deduction: calculate_plan_type_of_deduction(row.fetch("Plan Type of Deduction")),
       amount: row.fetch("Amount")
     }
   end
 
+  def calculate_plan_type_of_deduction(value)
+    if value == "No data"
+      nil
+    else
+      value
+    end
+  end
+
+  def calculate_no_of_plans_currently_repaying(value)
+    if value == "No data"
+      nil
+    else
+      value
+    end
+  end
+
```

Comment:
I'm not familiar in this area. can some confirm if this is desired? are there other instances of this that need to be addressed that are potentially missing coverage

Reply:
I'm not familiar on this bit of code, seems weird that a version upgrade would change this behaviour!

Reply:
for posterity this change issue was introduced by https://github.com/rails/rails/commit/366909d658e7050d1aea42ab372e85da622ba19c#diff-3f89af7ff219385718b7c0e4635cc4190476349db18f8bf80fb84fa1e7f5e289L240

adding the casting our side is by far a better option and we simply got lucky before it was doing our desired behaviour by default